bin/GameIndex.dbf

```diff
@@ -2244,6 +2254,8 @@ Serial = SCES-53409
 Name   = Sly 3
 Region = PAL-M11
 Compat = 5
+vuRoundMode = 0 // Fix physics issue in some levels.
```

Comment:
You need to be more specific here. Physics issue and some level aren't very clear. I know these settings solve a bomb mission in Episode 5, but is it required for other levels too?